### DISCUSSION TITLE
Is there any way to implement the selection of multiple bounding boxes?

### DISCUSSION THREAD

**Discussion by andrheapaltao** [annotorious/annotorious#282] [Q&A]
I would like to ask as stated in the title, if there's any way to implement the multiple selection of bounding boxes and update multiple bounding boxes all at once. I have not seen any sample of it, maybe it can be another feature for the library?

**Comment by rsimon** [comment]
Sorry, that's not possible at the moment. But, yes, it will be supported in the next major release of Annotorious (version 3).

**Reply by andrheapaltao** [reply]
Oh wow. When is the estimate date of release?

**Reply by rsimon** [reply]
It's still going to take some time. There's no estimated release date yet, I'm afraid. 

Work is ongoing in public here: https://github.com/annotorious/annotorious-v3. But at the moment this is a re-write of the core only. It will easily take until early 2024 until this reaches feature parity with v2.7.x. But we'll get there eventually ;-)

**Reply by andrheapaltao** [reply]
I see. That's good to know! Thank you so much!
